google/ggrc-core
# GGRC-3846 Cannot edit Custom roles fields in modal windows

# Issue description

Cannot edit Custom roles fields in modal windows

# Steps to test the changes

1. My Work page > Assessment tab
2. Invoke Edit modal window
3. Hover over Custom role field or Assignee/Verifier/Creator fields: edit button is not displayed

**Actual Result:** Cannot edit Custom roles fields in modal windows
**Expected Result:** User should edit Custom roles fields in modal windows
**NOTE:** Reproduced for all objects

# Solution description

Regression issue - #6732. 
Add missed imports. 

# Sanity checklist




# PR Review checklist
